<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="edit_min_firstleg">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-80</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_min_secondleg">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_min_thirdLeg">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-90</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_max_firstLeg">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>80</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_max_secondLeg">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>90</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_max_thirdLeg">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>110</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>90</string>
    </property>
   </widget>
   <widget class="QSlider" name="tbr_firstLeg">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>8000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="tbr_secondLeg">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="tbr_thirdLeg">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="edit_firstLeg">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="edit_secondLeg">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="edit_thirdLeg">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lv_legs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_send">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_send">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>175</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_load">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_offset">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>offset 적용해제</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_sign">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>offset 부호변경</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>306</width>
      <height>285</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="cb_timer">
         <property name="text">
          <string>&amp;Timer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="tbr_delay">
         <property name="minimum">
          <number>500</number>
         </property>
         <property name="maximum">
          <number>5000</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Edit_timerDelay">
         <property name="placeholderText">
          <string>1000</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="lv_commands"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="edPath"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="btn_commands">
             <property name="text">
              <string>&amp;Load commands</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnClear">
             <property name="text">
              <string>&amp;Clear</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="cb_firstLeg">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>96</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>1번 모터</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_secondLeg">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>96</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>2번 모터</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_thirdLeg">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>96</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>3번 모터</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_home">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>180</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>긴급정지</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>911</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
